FizzBuzz - single parameter.rb

```diff
@@ -0,0 +1,50 @@
+def fizz_buzz
+
+  puts "Pick a number:"
+  is_fb = gets.chomp
+  is_fb = is_fb.gsub(/,/, '.')
+
+  def print_fizz_buzz(is_fb)
```

Comment:
Glad you extracted the core logic into the method. Please move it outside the `fizz_buzz` method to avoid defining method in a method. You will be able to use `print_fizz_buzz` as long as it's defined above the `fizz_buzz.

Reply:
Thanks, on it!